package.json

```diff
@@ -52,6 +52,7 @@
   ],
   "dependencies": {
     "bigi": "^1.4.0",
+    "bech32": "git@github.com:bitcoinjs/bech32.git#81bc22761b3c8d286d6d8dc7bc10bda855adeb78",
```

Comment:
Needs to be `https` rather than `git@` in order to check out for non-members of `bech32`